docs/release-notes.rst

```diff
@@ -0,0 +1,105 @@
+.. _release-notes
+
+Release Notes
+=============
+
+v1.0.0
```

Comment:
I think we're planning to do a splash `v1.0.0` release at some point in the future, so I'd vote for `v0.x.0` for some value of `x` less than 7 or so.